packages/core/src/components/pages/CustomFieldForm/CustomFieldForm.tsx

```diff
@@ -91,8 +91,8 @@ const CustomFieldForm = ({
   }, [fields, panels, readonly]);
 
   const attrsValues = useMemo(
-    () => (object?.id !== modelId ? null : getAttrsValue(object)),
-    [modelId, object],
+    () => (object?.id != modelId ? null : getAttrsValue(object, fieldType)),
```

Comment:
both object?.id and modelId are numbers, it should be compared with !==, and not just !=. Why did you change this ?

Reply:
the modelId is not number, it's a string 
https://github.com/axelor/axelor-mobile/pull/550/files/68cfd7053731d690681d3650bdd81ea1a5ea2d2d#diff-104a3b29f081c9e352c7baa5db1fc4450e5aadefb49e54648f2ad3077d3d2a92R47